lib/DDG/Meta/Helper.pm

```diff
@@ -64,6 +65,17 @@ Use booleans true and false to set options.
 
     $stash->add_symbol('&true', sub { 1 });
     $stash->add_symbol('&false', sub { 0 });
+
+=keyword rand_int
+
+generates random integer between 0 and supplied max integer (max defaults to 1).
+
+=cut
+
+	$stash->add_symbol('&rand_int', sub { 	
+		my $max = shift || 1;
```

Comment:
The more I think about it, the more I think it doesn't matter so long as it's documented clearly. I was going to go with 100 because it was the closest metaphor for how rand does 0-1 fixed point rand by default.